old-backup/fb-solutions-denormalized-data.md

```diff
@@ -0,0 +1,52 @@
+<!-- This content was written originally for the Concepts homepage.
```

Comment:
Like it says on the tin, this content is for the next PR.
It was written during work on this PR whilst conceptualising information.